README.md

```diff
@@ -1,11 +1,11 @@
 go.strava
 =========
 
-Go.strava provides a complete, **including upload,** wrapper library for the [Strava V3 API](http://strava.github.com/api).
+Go.strava provides a complete, **including upload,** wrapper library for the [Strava V3 API](https://developers.strava.com).
```

Comment:
The correct link to API page is:
https://developers.strava.com/docs/reference/

Reply:
Done, previously it was done intentionally (the URL for the main page), but contextually it's better to use reference page.
Thanks for the remark!